exercises/practice/gigasecond/create_test_table.sql

```diff
@@ -0,0 +1,25 @@
+DROP TABLE IF EXISTS tests;
+CREATE TABLE IF NOT EXISTS tests (
+    -- uuid and description are taken from the test.toml file
+    uuid TEXT PRIMARY KEY,
+    description TEXT NOT NULL,
+    -- The following section is needed by the online test-runner
+    status TEXT DEFAULT 'fail',
+    message TEXT,
+    output TEXT,
+    test_code TEXT,
+    task_id INTEGER DEFAULT NULL,
+    -- Here are columns for the actual tests
+    moment TEXT NOT NULL,
+    result TEXT NOT NULL
+);
+
+-- Note: the strings below contain literal tab, newline, carriage returns.
+
+INSERT INTO tests (uuid, description, moment, result)
+    VALUES
+        ('92fbe71c-ea52-4fac-bd77-be38023cacf7','date only specification of time','2011-04-25','2043-01-01T01:46:40'),
```

Comment:
Could we get spaces after commas? Or should I just ignore those then revamp them when we have a generator?